test/deprecated/illegalArgumentErrorInit.chpl

```diff
@@ -0,0 +1,7 @@
+const iae1 = new IllegalArgumentError(info="bad argument!"),
```

Comment:
Why doesn't this line trip the deprecation warning?

Reply:
I believe it's because I put a `pragma "last resort"` on that overload (without it I was getting ambiguity errors).

Reply:
That seems like a bug in the `deprecated` implementation, but that doesn't need to hold this PR up.